account_credit_control/models/credit_control_policy.py

```diff
@@ -0,0 +1,390 @@
+# Copyright 2012-2017 Camptocamp SA
+# Copyright 2017 Okia SPRL (https://okia.be)
+# Copyright 2020 Manuel Calero - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError, ValidationError
+
+CHANNEL_LIST = [("letter", "Letter"), ("email", "Email"), ("phone", "Phone")]
+
+
+class CreditControlPolicy(models.Model):
+    """ Define a policy of reminder """
+
+    _name = "credit.control.policy"
+    _description = """Define a reminder policy"""
+
+    name = fields.Char(required=True)
+    level_ids = fields.One2many(
+        comodel_name="credit.control.policy.level",
+        inverse_name="policy_id",
+        string="Policy Levels",
+    )
+    do_nothing = fields.Boolean(
+        help="For policies which should not " "generate lines or are obsolete"
```

Comment:
```suggestion
        help="For policies which should not generate lines or are obsolete"
```
Search for " " occurrences throughout the code and remove them in unnecessary cases like this, there are about five.